bench/tool/tool.ml

```diff
@@ -49,6 +49,8 @@ let wait_pid =
         Unix.sleepf timeout;
         did_timeout := true;
         (* we kill the process group id (pgid) which should be equal to pid *)
+        Unix.kill (-pid) 15;
+        Unix.sleepf 1.;
```

Comment:
We could do something like:

```ocaml
if Env.is_set "LOG_QUERY_PATH" then begin
   Unix.kill (-pid 15);
   Unix.sleepf 1.
end
```

So that it runs normally with the correct timeout when we don't log queries ?